<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="lbl_topic">
     <property name="font">
      <font>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Bildvergleich</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_a">
     <item>
      <widget class="QListWidget" name="lw_doublets">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="la_vertical_layout">
       <item>
        <layout class="QGridLayout" name="la_grid_layout">
         <item row="5" column="1">
          <widget class="QLabel" name="lbl_img_two_path_value">
           <property name="text">
            <string>Pfad von Bild 2</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbl_img_one_label">
           <property name="text">
            <string>Bild 1</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="lbl_img_two_path_label">
           <property name="text">
            <string>Pfad</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lbl_img_one_path_label">
           <property name="text">
            <string>Pfad</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lbl_img_two_label">
           <property name="text">
            <string>Bild 2</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QRadioButton" name="rb_img_one">
           <property name="text">
            <string>Löschen</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="lbl_img_one_path_value">
           <property name="text">
            <string>Pfad von Bild 1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="lbl_img_one_container">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="lbl_img_two_container">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QRadioButton" name="rb_img_two">
           <property name="text">
            <string>Löschen</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_b">
     <item>
      <widget class="QPushButton" name="btn_confirm">
       <property name="text">
        <string>Übernehmen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_discard">
       <property name="text">
        <string>Keine Dublette</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
